.github/workflows/opensearch_changelog_workflow.yml

```diff
@@ -0,0 +1,23 @@
+name: OpenSearch Changelog Workflow
+
+on:
+  pull_request_target:
+    types: [opened, reopened, edited]
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  update-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Parse changelog entries and submit request for changset creation
+        uses: BigSamu/OpenSearch_Changelog_Workflow@main
```

Comment:
Please version the action and release it for the entire community to use and contribute to. This should reference a specific version.

Reply:
@AMoo-Miki, I just add the tag `1.0.0-alpha1` for the release in the action. Here the [link](https://github.com/BigSamu/OpenSearch_Changelog_Workflow/tags). Now the workflow action is referring to:

```yml
...
 - name: Parse changelog entries and submit request for changset creation
        uses: BigSamu/OpenSearch_Changelog_Workflow@1.0.0-alpha1
...
```

Is this OK?

The action itself is still under my repo. I am waiting for instructions from your side to transfer the action repo to OpenSearch